lib/jormungandr/src/Cardano/Wallet/Jormungandr/Api.hs

```diff
@@ -130,6 +127,8 @@ instance Accept JormungandrBinary where
 instance FromBinary a => MimeUnrender JormungandrBinary a where
     mimeUnrender _ bs = Right $ runGet get bs
 
+instance MimeRender JormungandrBinary (Tx, [TxWitness]) where
+    mimeRender _ _bs = undefined
```

Comment:
Hmmm :thinking: ?

Reply:
I can remove the `PostSignedTx` endpoint from the `type Api = ...` definition, if that's better.

This is needed when constructing the client.

Reply:
I'd say that generally speaking, yes, if something isn't ready to be tested, don't include it _yet_. It's better than having a half-functioning API.